apps/console/src/features/organizations/components/organization-list.tsx

```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's not update the license year of the older files.

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3235